modules/client-common/src/constants.ts

```diff
@@ -375,17 +375,17 @@ export const LIVE_CONTRACTS: {
       ensRegistryAddress: activeContractsList.baseGoerli.ENSRegistry,
     },
     [SupportedNetwork.LOCAL]: {
-      daoFactoryAddress: "",
-      pluginSetupProcessorAddress: "",
-      multisigRepoAddress: "",
-      adminRepoAddress: "",
-      addresslistVotingRepoAddress: "",
-      tokenVotingRepoAddress: "",
-      multisigSetupAddress: "",
-      adminSetupAddress: "",
-      addresslistVotingSetupAddress: "",
-      tokenVotingSetupAddress: "",
-      ensRegistryAddress: "",
+      daoFactoryAddress: "0x4ed7c70F96B99c776995fB64377f0d4aB3B0e1C1",
```

Comment:
Definetly not the ideal solution harcoding a bunch of addresses in the sdk. Yo can overrid tthis addresses when you pass the context params